Data/market-hours/market-hours-database.json

```diff
@@ -10225,7 +10225,81 @@
       "saturday": [],
       "holidays": [
         "4/14/2017"
-      ]
-    }
+      ]    
+    },
+        "Forex-bitfinex-[*]": {
+            "dataTimeZone": "UTC",
+            "exchangeTimeZone": "UTC",
+            "sunday": [
+                {
+                    "start": "00:00:00",
+                    "end": "1.00:00:00",
```

Comment:
What happens when either a `MarketOnOpenOrder` or `MarketOnCloseOrder` is submitted if the exchange is always open?